CONTRIBUTING.md

```diff
@@ -3,7 +3,10 @@
 ## Quick guide
 
  * Fork the repo.
- * Checkout the branch you want to make changes on.
+ * Checkout the branch you want to make changes on:
+  * Previous release branch if you are fixing a bug for an existing feature (or the current release branch if the feature you are fixing was introduced in that version)
+  * Current release branch if you are adding a new feature
+  * Master if you are adding a new feature or make changes to the code that are not backward compatible
```

Comment:
so... for new feature we could go in both current release and master ? master is for BC breakers